Documentation/ApiOverview/MountPoints/Index.rst

```diff
@@ -1,174 +1,253 @@
 
-      *  About us
 
-MountPoint Configuration:
+Multisites
+==========
 
-:Title:
- Basic MountPoint
-:URL Segment:
- basic-mountpoint
-:Target Page:
- Company
-:Mount Point Display Option:
- "Show this page"
+((Make this paragraph simpler or more expressive or drop it.)) In cross domain
+sites the rendering context of each site is kept while only *the content* is
+retrieved from somewhere else. This ensures that the reader will never notice
+that content is reused and retrieved from a different site or a different
+branch of the page tree.
 
-Generated URLs:
 
-- `https://example.com/basic-mountpoint` -> shows content on this page
-- `https://example.com/basic-mountpoint/about-us` -> shows `company/about-us` content
-- `https://example.com/company/`
-- `https://example.com/company/about-us`
+Considerations and limitations
+==============================
 
-Basic MountPoint - "Show the mounted page"
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. ordered just alphabetically as there is no hierarchy
 
-Example Pagetree:
+Duplicated content and SEO
+   Mount points create "duplicate content" by definition. This may work against
+   search engine optimisation (SEO) and may need extra consideration.
 
+Menus and linking to mounted subpages
+   Think of it: Mounted pages don't have a representation of its own in the
+   page tree. This also means that you cannot have links to mounted subpages.
+   Instead menus can be used to make mounted subpages accessable. TYPO3 takes
+   care of the difficult task of proper menu generation.
 
-*  Root
+Multi language support
+   This is possible. All sites have to agree on the same meaning of language
+   IDs however.
 
-   *  Basic MountPoint
-   *  Company
+Slug uniqueness
+   Attention: *Slug uniqueness* is NOT garanteed for *multisites*.
 
-      *  About us
+   Here is an example to illustrate the reason: Let's say a mount point page
+   has the slug `/more` and mounts a page that has a subpage `imprint`. Let's
+   further assume the mount page has a sibling `more/imprint` which is a
+   regular page. Then there is a collision of a type that TYPO3 cannot detect.
+   The regular page will always work but the mounted subpage will never be
+   shown.
 
-MountPoint Configuration:
+   ((**TODO:** Though I've rewritten this passage I have to confess that I
+   don't really understand this just by reading - marble.))
 
-:Title:
- Basic MountPoint
-:URL Segment:
- basic-mountpoint
-:Target Page:
- Company
-:Display Option:
- "Show the mounted page"
 
-Generated URLs:
+Examples
+========
 
-- `https://example.com/basic-mountpoint` -> shows `company` content
-- `https://example.com/basic-mountpoint/about-us` -> shows `company/about-us` content
-- `https://example.com/company/`
-- `https://example.com/company/about-us`
+Example: Simple usage
+---------------------
 
-Multi-Site support
--------------------
+*Intent:* Show an existing part of the page tree a second time at another
```

Comment:
I left out these explanations here on purpose, as the intent stated is basically the intent to use mount points in general, independent of the concrete example.

Reply:
I would read this difference in our sentences this way: You're adressing specifically insiders and developers - I was thinking that anybody interested in reading about mount point should have a chance to understand.

Reply:
TYPO3 Explained is currently content-wise mostly targeted at developers or integrators, it's not beginner-friendly. If we wanted to address both concerns, we should / would need to add an explanation of mount points to the editor guide (where someone already noted in slack that it's more an "introduction to the backend") - in the context of TYPO3 explained I'm trying to be concise rather than beginner-friendly.